server/src/test/java/org/opensearch/plugins/ClusterPluginTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugins;
+
+import org.opensearch.cluster.node.DiscoveryNode;
+import org.opensearch.test.OpenSearchSingleNodeTestCase;
+
+import java.util.Collection;
+
+public class ClusterPluginTests extends OpenSearchSingleNodeTestCase {
+
+    @Override
+    protected Collection<Class<? extends Plugin>> getPlugins() {
+        return pluginList(DummyClusterPlugin.class);
+    }
+
+    public void testOnNodeStarted_shouldContainLocalNodeInfo() {
```

Comment:
Thanks for the tests, could you what about adding a test that confirms `onNodeStarted()` is called on plugins that aren't using the newer interface?